src/components/oculus-touch-controls.js

```diff
@@ -50,8 +51,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       button5: 'surface'
     },
     'right': {
-      axis0: 'thumbstick',
-      axis1: 'thumbstick',
+      axes: {'thumbstick': [0, 1]},
```

Comment:
I prefer to have `axis + index` instead of an array for consistency with buttons